<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>549</width>
    <height>198</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(32, 31, 49);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="1">
        <widget class="QLabel" name="label_4">
         <property name="styleSheet">
          <string notr="true">font: 700 9pt &quot;MS Sans Serif&quot;;
color: rgb(172, 173, 183);</string>
         </property>
         <property name="text">
          <string>SET MIN MS</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QSpinBox" name="spinBox_minMS">
         <property name="styleSheet">
          <string notr="true">font: 700 9pt &quot;MS Sans Serif&quot;;
background-color: rgb(0, 0 ,0);
color: rgb(0, 232, 0);</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QSpinBox" name="spinBox_maxMS">
         <property name="styleSheet">
          <string notr="true">font: 700 9pt &quot;MS Sans Serif&quot;;
background-color: rgb(0, 0 ,0);
color: rgb(0, 232, 0);</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="label_5">
         <property name="styleSheet">
          <string notr="true">font: 700 9pt &quot;MS Sans Serif&quot;;
color: rgb(172, 173, 183);</string>
         </property>
         <property name="text">
          <string>SET MAX MS</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 700 9pt &quot;MS Sans Serif&quot;;
color: rgb(172, 173, 183);</string>
         </property>
         <property name="text">
          <string>SERVO ANGLE</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QSpinBox" name="spinBox">
         <property name="styleSheet">
          <string notr="true">font: 700 9pt &quot;MS Sans Serif&quot;;
background-color: rgb(0, 0 ,0);
color: rgb(0, 232, 0);</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QPushButton" name="pushButton">
         <property name="styleSheet">
          <string notr="true">font: 700 9pt &quot;MS Sans Serif&quot;;
background-color: rgb(189, 206, 214);
color: rgb(47, 55, 77);</string>
         </property>
         <property name="text">
          <string>SEND ANGLE</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3" rowspan="3">
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="label_6">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="styleSheet">
              <string notr="true">font: 700 9pt &quot;MS Sans Serif&quot;;
color: rgb(172, 173, 183);</string>
             </property>
             <property name="text">
              <string>MAX MS</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLCDNumber" name="lcdNumber_maxMS">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(0, 232, 0);
background-color: rgb(0, 0, 0);</string>
             </property>
             <property name="frameShape">
              <enum>QFrame::NoFrame</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Sunken</enum>
             </property>
             <property name="digitCount">
              <number>4</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLabel" name="label_7">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="styleSheet">
              <string notr="true">font: 700 9pt &quot;MS Sans Serif&quot;;
color: rgb(172, 173, 183);</string>
             </property>
             <property name="text">
              <string>MIN MS</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLCDNumber" name="lcdNumber_minMS">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(0, 232, 0);
background-color: rgb(0, 0, 0);</string>
             </property>
             <property name="frameShape">
              <enum>QFrame::NoFrame</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Sunken</enum>
             </property>
             <property name="digitCount">
              <number>4</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_read">
           <property name="styleSheet">
            <string notr="true">font: 700 9pt &quot;MS Sans Serif&quot;;
background-color: rgb(189, 206, 214);
color: rgb(47, 55, 77);</string>
           </property>
           <property name="text">
            <string>READ MAX &amp;&amp; MIN</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="3" column="1" colspan="2">
        <widget class="QPushButton" name="pushButton_maxMS">
         <property name="styleSheet">
          <string notr="true">font: 700 9pt &quot;MS Sans Serif&quot;;
background-color: rgb(189, 206, 214);
color: rgb(47, 55, 77);</string>
         </property>
         <property name="text">
          <string>SET MAX &amp;&amp; MIN MS</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 700 9pt &quot;MS Sans Serif&quot;;
color: rgb(172, 173, 183);</string>
           </property>
           <property name="text">
            <string>ENGINE 2 POWER</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 700 9pt &quot;MS Sans Serif&quot;;
color: rgb(172, 173, 183);</string>
           </property>
           <property name="text">
            <string>ENGINE 1 POWER</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <property name="sizeConstraint">
          <enum>QLayout::SetDefaultConstraint</enum>
         </property>
         <item>
          <widget class="QSpinBox" name="spinBox_2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 700 9pt &quot;MS Sans Serif&quot;;
background-color: rgb(0, 0 ,0);
color: rgb(0, 232, 0);</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBox_3">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 700 9pt &quot;MS Sans Serif&quot;;
background-color: rgb(0, 0 ,0);
color: rgb(0, 232, 0);</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_2">
         <property name="styleSheet">
          <string notr="true">font: 700 9pt &quot;MS Sans Serif&quot;;
background-color: rgb(189, 206, 214);
color: rgb(47, 55, 77);</string>
         </property>
         <property name="text">
          <string>SEND POWER</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
